### PR TITLE
[Azure Stack] Update the admin disk API to support new features

### PR DESCRIPTION
**Pull request by zyxyoshine**
Why
1. Since the disks (snapshots) on the Refs volume may share extends, the actual size property of the current get-disk admin API cannot tell the user how much size can be freed on the source volume if the disk is migrated to another volume.
2. The current get-disk API does not include creation data, so it is difficult to correlate snapshots with the original disk.

How
1. Add a new switch parameter showSizeDetail in the admin API, which indicates whether to query the exclusive allocated size from blob service and return to client.
2. Add the creation data to the get-disk API.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [x] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  Azs build 2101. 
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [x] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://msazure.visualstudio.com/One/_workitems/create/Scenario?templateId=d1c215ec-5d6a-4cb3-a193-95ddd0a1ba96&ownerId=85bb92fb-21fe-41ca-9309-47f2c5f57515). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by zyxyoshine** [comment]
@bganapa please help review.

**Comment by bganapa** [comment]
Please note that the changes are being made in a preview api version.

**Review by bganapa — APPROVED** [review]

**Comment by zhenglaizhang** [comment]
@zyxyoshine pls help resolve the conflicts

**Comment by zyxyoshine** [comment]
@zhenglaizhang Thanks, resolved the conflicts.

**Comment by zyxyoshine** [comment]
@bganapa @zhenglaizhang any update?
Our plan is to release with Azs build 2101.

**Comment by zyxyoshine** [comment]
And please help remove the no-recent-activity label.

**Comment by bganapa** [comment]
@zhenglaizhang  Could you pls let us know what is needed here to get this merged? It is marked as Wait for ARM feedback.

**Comment by zhenglaizhang** [comment]
@bganapa If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them.

**Comment by zyxyoshine** [comment]
@bganapa This PR was again marked as no-recent-activity. Do we have contract with ARM team?

**Comment by jorgecotillo** [comment]
You need to get a breaking change exception, please follow this [link](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes), the reason is because your preview api is more than 12 months old and exceeded the recommended lifespan of a preview api, like mentioned [here](https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html#preview-apis).

File: specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json
Comment (RamyasreeChakka) [inline specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json:165]: Please note adding new properties to existing API version is considered as a breaking change as the previously generated SDKs will not have these properties. Ideally need to introduce a new API version. 
I would recommend to expose a new API version as the current API version exists from a long time 2018-03-01-preview and kind of qualify as a GA version for customers.

Comment (RamyasreeChakka) [reply specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json:165]: Please refer https://aka.ms/bcforapi

Comment (zyxyoshine) [reply specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json:165]: sure, we will introduce a new api version for this changes.
@FireDefend  FYI.

**Pull request closed without merging**
